pyrtl/importexport.py

```diff
@@ -1435,7 +1436,7 @@ def output_to_verilog(
         bitwidth=8, reset_value=4)`` generates Verilog like ``reg[7:0] foo = 8'd4;``.
     :param block: Block to be walked and exported. Defaults to the :ref:`working_block`.
     """
-    _VerilogOutput(block, add_reset).output_to_verilog(dest_file, initialize_registers)
+    _VerilogOutput(block, add_reset).output_to_verilog(dest_file, initialize_registers, module_name=module_name)
 
 
 def output_verilog_testbench(
```

Comment:
`output_verilog_testbench` also needs to be updated to support `module_name`, it currently assumes the instantiated module is named `toplevel`.